src/main/de/linearbits/preferences/PreferenceBoolean.java

```diff
@@ -37,11 +51,20 @@ public PreferenceBoolean(String label) {
 
     @Override
     protected Editor<Boolean> getEditor() {
-        return new EditorBoolean(getDialog(), getDefault());
+        return new EditorBoolean(getDialog(), getDefault(),this.hideLabel);
```

Comment:
There is a blank missing before this.hideLabel. You could use auto-format to fix this (hint: apply to modified lines only).